sources/packages/web/src/components/common/AssessmentAwardDetails.vue

```diff
@@ -248,15 +248,15 @@ export default defineComponent({
       () =>
         !!(
           isFirstDisbursementCompleted.value &&
-          props.assessmentAwardData.finalAward.disbursementReceipt1Id
+          !!props.assessmentAwardData.finalAward
         ),
     );
 
     const showSecondFinalAward = computed<boolean>(
       () =>
         !!(
           isSecondDisbursementCompleted.value &&
-          props.assessmentAwardData.finalAward.disbursementReceipt2Id
+          props.assessmentAwardData.finalAward
```

Comment:
Why we are not changing the variable as a number? is this because undefined/null can come?

Reply:
`finalAward` is an object that contains all the properties for all the awards.
For instance:
```ts
 finalAward: {
    disbursementReceipt1cslf: 1000,
    disbursementReceipt1csgp: 10001,
    disbursementReceipt1bcsl: 1005,
    disbursementReceipt1bcag: 1006,
    disbursementReceipt1bgpd: 1007,
    disbursementReceipt1sbsd: 1008,
    disbursementReceipt2cslf: 1000,
    disbursementReceipt2csgp: 10001,
    disbursementReceipt2bcsl: 1005,
    disbursementReceipt2bcag: 1006,
    disbursementReceipt2bgpd: 1007,
    disbursementReceipt2sbsd: 1008,
}
```